### DISCUSSION TITLE
Release 0.5.0 and 0.5.1 fail

### DISCUSSION THREAD

**Discussion by LucasCano** [flybywiresim/aircraft#2927] [Q&A] [answered]
Hi,
With releases 0.5.0 & 0.5.1 after activating the batteries, none of the panels turn on. When starting the engines with Ctrl + E, all screens are blinking. It is not possible to use avionics.
Now I'm using the realese 0.4.2, it takes a while to turn on the panels, but eventually they turn on.

Any ideas about it? Thanks

using MSFS2020 v. 1.11.6

**Comment by ChruutvoLuzi** [comment]
Are you sure you are using MSFS Version 1.11.6? The latest should be 1.12.13.0, maybe that causes the issues

**Comment by Inaxair** [comment, marked as answer]
Sorry, we cannot support users with pirated version of the aircraft.

Inaxair from FlyByWire Simulations
